mars/deploy/oscar/session.py

```diff
@@ -41,6 +41,7 @@
 from ...services.lifecycle import AbstractLifecycleAPI, LifecycleAPI
 from ...services.meta import MetaAPI, AbstractMetaAPI
 from ...services.session import AbstractSessionAPI, SessionAPI
+from ...services.mutable.api.oscar import MutableAPI
```

Comment:
You can import the `MutableAPI` for the `services.mutable` module, just like the other lines.